Gaijinizer patch/CommonEvents_679.txt

```diff
@@ -85,30 +85,30 @@ What a cute little boy...♡#DL
 
 ボウヤの恥ずかしいところ…いっぱい見せて♡
 > CONTEXT: CommonEvents_679/36/Dialogue/enemy33
-Will you give in and masturbate in front
+Now, will you surrender and masturbate in front
```

Comment:
I'm assuming Kazuya's already given in to her, so it could also be translated as "will you submissively masturbate..."